### PR TITLE
Limit `expand-all-hidden-comments` to 200

### LINKED ISSUE #2931: `expand-all-hidden-comments` doesn't work on super-long issues
I'm attempting to expand all 671 comments found in this issue: https://github.com/Koenkk/zigbee2mqtt/issues/1429

I have the Refined Github extension v20.3.16 added to Google Chrome v80.0.3987.149 on MacOS 10.15.3, and I have added a personal token within Options.

When I Option+Click the "Load More..." link, it very quickly disappears and just shows the first of the latest set of comments from 8 days ago. e.g.

image

After trying Opt+Click:
image

This is the related pull request: https://github.com/sindresorhus/refined-github/pull/2754

TIA

### PR DESCRIPTION
**Pull request by yakov116**
Fixes #2931

Just saw your comment,
>What we can do, if easy, is to limit it to ~200 items

Should I 200?
<!--

Thanks for contributing! 🍄

1. LINKED ISSUES:
   Does this PR close/fix an existing issue? Write something like `Closes #10`

2. TEST URLS:
   Add some test URLs

3. SCREENSHOT:
   Add a screenshot here if your PR makes visual changes

### CODE REVIEW

File: source/features/expand-all-hidden-comments.ts [hunk 1]
```diff
@@ -13,7 +13,11 @@ function handleAltClick(event: DelegateEvent<MouseEvent, HTMLButtonElement>): vo
 	}
 
 	const form = event.delegateTarget.form!;
-	const hiddenItemsCount = looseParseInt(form.textContent!);
+	let hiddenItemsCount = looseParseInt(form.textContent!);
+	if (hiddenItemsCount > 250) { // https://github.com/sindresorhus/refined-github/issues/2931
+		hiddenItemsCount = 250;
+	}
```
Comment (fregante) [inline source/features/expand-all-hidden-comments.ts]:
```suggestion
	const hiddenItemsCount = Math.min(
		250, // https://github.com/sindresorhus/refined-github/issues/2931
		looseParseInt(form.textContent!)
	);
```

Comment (yakov116) [reply source/features/expand-all-hidden-comments.ts]: Cool never knew that!

**Comment by fregante** [comment]
Can you also update the description and readme?

```diff
- On long discussions where GitHub hides comments under a "Load more...", alt-clicking it will load *all* the comments at once instead of part of them.
+ On long discussions where GitHub hides comments under a "Load more...", alt-clicking it will load up to 200 comments at once instead of 60.
```

**Comment by fregante** [comment]
# 🌮

**Comment by b-studios** [comment]
Has this change been reverted? On an issue with 140 hidden items I alt-click (Mac OS X) and it only expands 60 items.

**Comment by yakov116** [comment]
Working fine for me on windows

**Pull request merged**